be/src/http/action/stream_load.cpp

```diff
@@ -489,6 +489,15 @@ Status StreamLoadAction::_process_put(HttpRequest* http_req, StreamLoadContext*
             return Status::InvalidArgument("Invalid load_dop format");
         }
     }
+    if (!http_req->header(HTTP_ENABLE_REPLICATED_STORAGE).empty()) {
```

Comment:
How about the performance gained?

Reply:
performance gained for primary key replicated storage?

Reply:
Yes. The performance gained between replicated storage and non-replicated storage for primary-key table

Reply:
In theory, it is the same as other tables, and the specific results will be supplemented in the performance test of version 2.5.